models/request.js

```diff
@@ -0,0 +1,74 @@
+var User = require('./user');
+var Service = require('./service');
+module.exports = function(sequelize, DataTypes) {
+    var Request = sequelize.define("Request", {
+      recipient_id: {
+        type: DataTypes.INTEGER,
+        primaryKey:true,
+        allowNull: false,
+        autoIncrement: true,
+        unique: true
+      },
+      services_id: {
+        type: DataTypes.INTEGER,
+        primaryKey:true,
+        allowNull: false,
+        autoIncrement: true,
+        unique: true
+      },
+      finalized: {
+        type: DataTypes.BOOLEAN,
+        primaryKey:true,
+        allowNull: true,
+      },
+      detail: {
+        type: DataTypes.TEXT,
+        allowNull: false,
+        validate: {
+          len: [1]
+        }          
+      },
+      requester_rating: {
+        type: DataTypes.INTEGER,
+        primaryKey:true,
+        allowNull: true,
+        autoIncrement: true,
+        unique: true
+      },
+      bounty_points: {
+        type: DataTypes.INTEGER,
+        primaryKey:true,
+        allowNull: true,
+        autoIncrement: true,
+        unique: true
+      },
+      start_date: {
+          type: DataTypes.DATE,
+          allowNull: false
+      },
+      end_date: {
+          type: DataTypes.DATE,
+          allowNull: false
+      }
+    });
+    User.associate = function(models) {
+        // We're saying that a Post should belong to an Author
+        // A Post can't be created without an Author due to the foreign key constraint
+        Post.belongsTo(models.User, {
+          foreignKey: {
+            allowNull: false
+          }
+        });
+      };
+    Service.associate = function(models) {
+        // We're saying that a Post should belong to an Author
+        // A Post can't be created without an Author due to the foreign key constraint
+        Post.belongsTo(models.Service, {
+          foreignKey: {
+            allowNull: false
+          }
+        });
+      };
```

Comment:
These will need to be removed.

Things to note:
- We only want to deal with `Request` `source` associations in this file
- `Post` isn't a part of our project

After looking it up and trying things out, we will want this instead:

```JavaScript
	Requests.associate = function(models) {
		Requests.belongsTo(models.Users);
		Requests.belongsTo(models.Services);
		Requests.hasMany(models.Responses);
	};
```

This will allow us to get the recipient